### ISSUE TITLE
ClassGraph#verbose() causes error in jlink'd distribution

### ISSUE THREAD

**Issue by rendaw** [classgraph/classgraph#313]
First it causes this at program launch:
```
$ ./test1/target/jlink-image/bin/java -p modules -m test1/test1.Main
Exception in thread "main" java.lang.NoClassDefFoundError: java/util/logging/Logger
	at io.github.classgraph@4.6.18/nonapi.io.github.classgraph.utils.LogNode.<clinit>(LogNode.java:51)
	at io.github.classgraph@4.6.18/io.github.classgraph.ClassGraph.verbose(ClassGraph.java:103)
	at test1/test1.Main.main(Main.java:12)
Caused by: java.lang.ClassNotFoundException: java.util.logging.Logger
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:583)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:521)
	... 3 more
```

Adding `requires java.logging;` to module-info.java keeps it in the distribution but actually using it causes this error:
```
$ ./test1/target/jlink-image/bin/java -p modules -m test1/test1.Main
Exception in thread "main" java.lang.IllegalAccessError: class nonapi.io.github.classgraph.utils.LogNode (in module io.github.classgraph) cannot access class java.util.logging.Logger (in module java.logging) because module io.github.classgraph does not read module java.logging
	at io.github.classgraph@4.6.18/nonapi.io.github.classgraph.utils.LogNode.<clinit>(LogNode.java:51)
	at io.github.classgraph@4.6.18/io.github.classgraph.ClassGraph.verbose(ClassGraph.java:103)
	at test1/test1.Main.main(Main.java:13)
```

**Comment by lukehutch** [comment]
Duplicate of #309, fixed in master. Thanks!

**Comment by lukehutch** [comment]
PS @rendaw you're currently using 4.6.18, which is about 15 releases behind master. Please test any other bugs you run into against the master version until any more issues are found and fixed. Thanks!

**Comment by lukehutch** [comment]
Released in 4.8.0.

https://github.com/classgraph/classgraph/releases/tag/classgraph-4.8.0

Thanks for the bug report!
